masu/database/sql/reporting_ocpusagelineitem_aggregates.sql

```diff
@@ -24,10 +24,10 @@ CREATE TEMPORARY TABLE reporting_ocpusagelineitem_aggregates_{uuid} AS (
         li.node,
         sum(li.pod_usage_cpu_core_seconds) / 3600 as pod_usage_cpu_core_hours,
         sum(li.pod_request_cpu_core_seconds) / 3600 as pod_request_cpu_core_hours,
-        max(li.pod_limit_cpu_cores) as pod_limit_cpu_cores,
+        sum(li.pod_limit_cpu_core_seconds) / 3600 as pod_limit_cpu_core_hours,
         sum(li.pod_usage_memory_byte_seconds / li.total_seconds) * 1e-9 as pod_usage_memory_gigabytes,
         sum(li.pod_request_memory_byte_seconds / li.total_seconds) * 1e-9 as pod_request_memory_gigabytes,
-        max(li.pod_limit_memory_bytes) * 1e-9 as pod_limit_memory_gigabytes
+        sum(li.pod_limit_memory_byte_seconds / li.total_seconds) * 1e-9 as pod_limit_memory_gigabytes
```

Comment:
I think we still want max here. this will be a sum of the limit value in all rows, which is not the limit as set on the pod.